slapp/explorer/views.py

```diff
@@ -544,23 +544,59 @@ class Results(TemplateView):
 
     template_name = "pages/results.html"
 
-    def get_context_data(self, **kwargs) -> dict:
-        """Manage context data."""
-        context = super().get_context_data(**kwargs)
-
-        context["home_url"] = reverse("explorer:home")
-        context["added_value_url"] = reverse("added_value:index")
-        return context
+    def dispatch(self, request: HttpRequest, *args: object, **kwargs: object) -> HttpResponseBase:
+        """Return electricity flow charts."""
+        if request.resolver_match.url_name == "flow_chart":
+            return flow_chart(request)
+        if request.resolver_match.url_name == "cost_capacity_chart":
+            return cost_capacity_chart(request)
 
-
-class Calculator(TemplateView):
-    """Display the Calculator page with value creation and the calculator."""
-
-    template_name = "pages/calculator.html"
+        return super().dispatch(request, *args, **kwargs)
 
     def get_context_data(self, **kwargs) -> dict:
         """Manage context data."""
         context = super().get_context_data(**kwargs)
 
-        context["next_url"] = reverse("explorer:home")
+        chart_data = {
+            "electricity": {
+                "categories": ["Jan", "Feb", "Mar"],
+                "series": {
+                    "Generation": [120, 200, 150],
+                    "Consumption": [90, 50, 110],
+                },
+            },
+            "heat": {
+                "categories": ["Jan", "Feb", "Mar"],
+                "series": {
+                    "Generation": [80, 60, 90],
+                    "Consumption": [40, 30, 55],
+                },
+            },
+            "capacity": {
+                "categories": ["Wind", "Solar", "Biomass"],
+                "series": {
+                    "Existing": [300, 200, 100],
+                    "Addition": [30, 20, 10],
+                },
+            },
+            "costs": {
+                "categories": ["Project A", "Project B", "Project C"],
+                "series": {
+                    "Variable costs": [50000, 30000, 45000],
+                    "Investment": [200000, 150000, 250000],
+                },
+            },
+        }
```

Comment:
This coudl also be moved into separate function, which can be changed later in order to load actual data.